network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/search_bar.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+<div class="tw-relative">
+  <label for="search" class="tw-sr-only">{% translate 'Search' context 'Button' %}</label>
+  <input type="text" value="{{ search_query }}" name="search" id="search" placeholder="{% translate placeholder_text context 'Search input' %}" class="tw-w-full form-control tw-h-auto tw-p-4 tw-pr-7 tw-rounded-none {{input_styles}}" />
```

Comment:
Hi @TheoChevalier! I was wondering if you could take a look and let me know if this is proper use of the translate block, as I am passing in the text as a parameter from "research_search_bar" and "blog_search_bar".

Thanks in advance!

Reply:
@danielfmiranda I would be surprised if this works, because you can pass any text dynamically to the template. I think you will need to move the translation into the template where the text is actually defined. There you can use `{% translate "My new placeholder text" context "Search input placeholder text" as placeholder_text %}` and then pass the `placeholder_text` variable to the included template, like so `{% include "wagtailpages/fragments/search_bar.html" with placeholder_text=placeholder_text input_styles="tw-border-gray-40" %}`